File: Modul 1. Variable, Tipe Data, dan Operator/PRAK104-2310817120003-Natalie_Grace_Katindagho/PRAK104-2310817120003-Natalie_Grace_Katindagho.py
a = 400000
b = 350000
disc_a = 13/100
disc_b = 21/100
harga_disc_a = int (a * disc_a)
harga_disc_b = int (b * disc_b)
harga_akhir_a = int (a - harga_disc_a)
harga_akhir_b = int (b - harga_disc_b)
print("Harga sepatu A adalah", a)
print("Harga sepatu B adalah", b)
print("Sepatu A mendapat diskon 13% sehingga harganya menjadi", harga_akhir_a)
print("Sepatu B mendapat diskon 21% sehingga harganya menjadi", harga_akhir_b)

File: Modul 1. Variable, Tipe Data, dan Operator/PRAK104-2310817120003-Natalie_Grace_Katindagho/PRAK104-2310817120003-Natalie_Grace_Katindagho.c
#include <stdio.h>

void main()
{
    int a = 400000;
    int b = 350000;
    float disc_a = 0.13;
    float disc_b = 0.21;
    float harga_disc_a = a * disc_a;
    float harga_disc_b = b * disc_b;
    int harga_akhir_a = a - harga_disc_a;
    int harga_akhir_b = b - harga_disc_b;
    printf("Harga sepatu A adalah %d\n", a);
    printf("Harga sepatu B adalah %d\n", b);
    printf("Sepatu A mendapat diskon 13%% sehingga harganya menjadi %d\n", harga_akhir_a);
    printf("Sepatu B mendapat diskon 21%% sehingga harganya menjadi %d\n", harga_akhir_b);
}